S23/fulvius31/app/src/main/java/lowercase/App.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package lowercase;
+
+public class App {
+
+    public static void main(String[] args) {
+        Lowercaser lowercaser = new Lowercaser("nÕg","ga-US");
```

Comment:
Instead of changing input in the code every time.can we just read it using scanner/buffer reader.

Reply:
I've just removed the useless code on main.